src/fides/data/sample_project/cookie_house/src/pages/index.tsx

```diff
@@ -4,19 +4,25 @@ import Head from 'next/head';
 
 import Home from '../components/Home';
 import pool from '../lib/db';
+import Script from 'next/script';
 
 interface Props {
   products: Product[];
+  gtm_key?: string;
 }
 
+
 export const getServerSideProps: GetServerSideProps<Props> = async () => {
   const results = await pool.query<Product>('SELECT * FROM public.product;');
-  return { props: { products: results.rows } };
+  return { props: { products: results.rows, gtm_key: process.env.GOOGLE_TAG_MANAGER_CONTAINER } };
```

Comment:
```suggestion
  const gtmRegex = /^[0-9a-zA-Z-]+$/
  if (process.env.GOOGLE_TAG_MANAGER_CONTAINER?.match(gtmRegex)) {
    let gtm_key = process.env.GOOGLE_TAG_MANAGER_CONTAINER
  }
  return { props: { products: results.rows, gtm_key } };
```

...or something like that

Reply:
Need to confirm what a GTM key would look like, but AFAIK it's something like `GA-123-456-ABC` so alphanumeric characters plus `-` should be safe.